ESP8266/ESP8266.h

```diff
@@ -265,20 +266,30 @@ class ESP8266
      */
     nsapi_connection_status_t get_connection_status() const;
 
+    /**
+     * Start board's and ESP8266's UART flow control
+     *
+     * @return true if started
+     */
+    bool start_uart_hw_flow_ctrl();
```

Comment:
Should this be private?
We are calling this, and not expecting users to call it? am I right?